.github/workflows/integration-test-workflow.yml

```
@@ -46,8 +46,7 @@ jobs:
         continue-on-error: true
 
       - name: purge
-        run: sudo dpkg -P tedge_agent tedge_mapper tedge_apt_plugin tedge_apama_plugin tedge mosquitto libmosquitto1 collectd-core
-        continue-on-error: true
+        run: sudo dpkg -P tedge_agent tedge_logfile_request_plugin tedge_mapper tedge_apt_plugin tedge_apama_plugin tedge mosquitto libmosquitto1 collectd-core
```

Please also purge: mosquitto-clients collectd purging fails (at least for me without them).
```suggestion
        run: sudo dpkg -P tedge_agent tedge_logfile_request_plugin tedge_mapper tedge_apt_plugin tedge_apama_plugin tedge mosquitto libmosquitto1 collectd-core mosquitto-clients collectd
```